### ISSUE TITLE
error buying weapon

### ISSUE THREAD

**Issue by donnayy** [Yeagorn/esx_secretshop#1]
When I buy the weapon i got a " Undefined " weapon on my inventory

**Comment by Yeagorn** [comment]
How did u define the weapon in the config?

**Comment by OneHypocrisy** [comment]
For me the gun is just not appearing in my inventory after i have purchased it, i haven't touched anything in the config.

**Comment by Yeagorn** [comment]
Have you tried setting isWeapon true in the config?

**Comment by OneHypocrisy** [comment]
i checked and it was already set to true

**Comment by Yeagorn** [comment]
Which inventory do you use?

**Comment by OneHypocrisy** [comment]
i use esx inventoryhud

**Comment by OneHypocrisy** [comment]
inventoryhud*

**Comment by Yeagorn** [comment]
Have you tried seeing in the weapon wheel if you have it ?

**Comment by OneHypocrisy** [comment]
yup i have, it doesn't appear in either after purchase. i try to restart the script and it just tells me i have the weapon already at the purchase prompt.

**Comment by Yeagorn** [comment]
Try setting isWeapon to false

**Comment by OneHypocrisy** [comment]
just tried and it didnt work and i dont know why, i really want to use this ugh haha

**Comment by Yeagorn** [comment]
any errors? I'll get into fixing it today so and give me your es_extended version.

**Comment by GoodGuyLeon** [comment]
im getting the same error? any fix for it yet?

**Comment by OneHypocrisy** [comment]
im just rocking  standard es_extended

**Comment by OneHypocrisy** [comment]
but no its still not working, tried some more last night and same error is occuring.

**Comment by Yeagorn** [comment]
Try using 1.1 until its fixed.
